### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by sawyer-nye**
I just noticed that v1 doesn't point anywhere when I was looking for the API documentation. Sorry for the incorrect branch name.

### CODE REVIEW

**Comment by jmm8046** [comment]
The fact that you made a PR for this honestly just made my day

**Comment by sawyer-nye** [comment]
> The fact that you made a PR for this honestly just made my day

ok boomer

**Comment by jmm8046** [comment]
But the README needs a few other things changed as well, I'm updating the live website rn, I'll push to this branch some more changes in a bit and then we can approve

File: README.md [hunk 1]
```diff
@@ -57,8 +57,8 @@ By default, ESLint will attempt to fix whatever errors possible; the rest should
 #### Links
 Resource | URL
 ------------ | -------------
- API | https://api.hackpsu.org/v1
- API Documentation | https://api.hackpsu.org/v1/doc
+ API | https://api.hackpsu.org/v2
```
Comment (jmm8046) [inline README.md]: So here, the v2 is not necessary (the link goes no where) so it should be just https://api.hackpsu.org/ (which displays a json if you go to it)

Comment (jmm8046) [reply README.md]: I'll fix it in my commit

File: README.md [hunk 2]
```diff
@@ -1,18 +1,18 @@
-# hackpsuS2018
-Repository for the main website for HackPSU Spring 2018
+##HackPSU Fall 2019
```
Comment (stan29308) [inline README.md]: This actually needs the space in front for the formatter. You can double check it if you click on the "Display rich diff" button in the top right.

File: README.md [hunk 3]
```diff
@@ -24,31 +24,33 @@ Api group | @apiGroup
...
-This repository is configured for Continuous Integration (CI) with Travis CI. Click the above Build Status link for more details on the last build. 
+This repository is configured for Continuous Integration (CI) with Travis CI. Click the above Build Status link for more details on the last build.
 The CI is programmed to run all tests for the API, and deploy the current version of the API to EB, as well as deploys the frontend from website directory to Firebase.
 All deployments happen on the _master_ branch. Therefore, as much as possible, commits should not be made directly to _master_. All development should be run on _dev_
-and then PRed into _master_. 
+and then PRed into _master_.
 
 To run the tests, run `npm test` in the main directory.
+To compile and run the main website, run `npm run watch-sass` in the main directory.
+To run anything from app.hackpsu.org, run `npm run start` in the user-registration-app directory.
```
Comment (stan29308) [inline README.md]: If you want these to be on separate lines, add __ to the end of the line.
https://stackoverflow.com/questions/24575680/new-lines-inside-paragraph-in-readme-md

File: README.md [hunk 4]
```diff
@@ -24,31 +24,33 @@ Api group | @apiGroup
...
 
 ## Testing
 
-Testing is handled by the CI platform Travis-CI. Tests are Javascript files under api/test. Tests should use the [MochaJS](http://Mochajs.org) testing framework. In addition, use 
+Testing is handled by the CI platform Travis-CI. Tests are Javascript files under api/test. Tests should use the [MochaJS](http://Mochajs.org) testing framework. In addition, use
 [ChaiJS](http://chaijs.com) for assertions. RESTful testing is handled by the [chai-http](http://chaijs.com/plugins/chai-http/) module.
```
Comment (stan29308) [inline README.md]: RESTful testing also uses the [mocha-typescript](https://www.npmjs.com/package/mocha-typescript) package.
Formatted using \[mocha-typescript\](https://www.npmjs.com/package/mocha-typescript)

File: README.md [hunk 5]
```diff
@@ -57,25 +59,15 @@ By default, ESLint will attempt to fix whatever errors possible; the rest should
...
- 1 | Volunteer
- 2 | Team Member
- 3 | Exec member
- 4 | Tech admin
-
-
- 
-  
+
+
```
Comment (stan29308) [inline README.md]: Is there a reason why these were removed?

File: README.md (hunk 2 above)
Comment (stan29308) [reply README.md]: Probably also only one hashtag

Comment (stan29308) [reply README.md]: Probably also only needs one hashtag

File: README.md (hunk 5 above)
Comment (jmm8046) [reply README.md]: I felt they were not relevant to the frontend, do you disagree?

File: README.md (hunk 2 above)
Comment (jmm8046) [reply README.md]: Fixed

File: README.md (hunk 4 above)
Comment (jmm8046) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (jmm8046) [reply README.md]: Done

File: README.md (hunk 2 above)
Comment (jmm8046) [reply README.md]: Done

Comment (jmm8046) [reply README.md]: I already said that lmao

**Pull request merged**